### PR TITLE
feat: toast for not owning items in wearable collection

### PR DESCRIPTION
**Pull request by dalkia**
## What does this PR change?

Shows a toast when a player doesn't own any items from a Linked Wearable collection

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=chore/toast-doesnt-own-item-wearble-collection
2. Open the backpack
3. Check that the toast is visible when selecting an item from the dropdown that you dont own any item of that collection.
4. Check that the toast closes when you click anywhere
5. Check that if you dont click anywhere, the toast fades out after 5 seconds

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by dalkia** [comment]
Not sure if this is the best way to do it. 

I saw that I could use the `NotificationSystem`; but it was too complicated to get the notification out of the parented `RectTransform` object that the notification systems put all notifications in. If we could reposition the toast, the notification system could be used.

**Comment by dalkia** [comment]
Video Reference

https://user-images.githubusercontent.com/1999557/179643462-6ed0160b-a1a6-4f03-8ae2-5a06f280d3bc.mp4

**Comment by RominaMarchetti** [comment]
@dalkia I talked with Álvaro and he told me that he was giving you some feedback in order to improve the animation but the final result is not good. Please, implement the original design keeping the current fade animation. Later, the UI animator will improve it.

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs [hunk 1]
```diff
@@ -128,6 +128,12 @@ public class AvatarEditorWearableFilter
     [SerializeField] internal TMP_Text sectionTitle;
     [SerializeField] internal GameObject avatarSection;
     [SerializeField] internal GameObject emotesSection;
+    
+    [Header("Click BLocker")]
+    [SerializeField] internal UIHelper_ClickBlocker blocker;
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs:133]: `clickBlocker` would fit a better name

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs [hunk 2]
```diff
@@ -128,6 +128,12 @@ public class AvatarEditorWearableFilter
     [SerializeField] internal TMP_Text sectionTitle;
     [SerializeField] internal GameObject avatarSection;
     [SerializeField] internal GameObject emotesSection;
+    
+    [Header("Click BLocker")]
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs:132]: i dont think its necessary to have a header here

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs [hunk 3]
```diff
@@ -128,6 +128,12 @@ public class AvatarEditorWearableFilter
...
+    [SerializeField] internal UIHelper_ClickBlocker blocker;
+    
+    [Header("Notifications")]
```
Comment (lorux0) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/AvatarEditorHUD/Scripts/AvatarEditorHUDView.cs:135]: i dont think its necessary to have a header here

**Review by lorux0 — APPROVED** [review]

**Review by RominaMarchetti — APPROVED** [review]

**Pull request merged**